test/public/lhcFills/detail.test.js

```diff
@@ -152,7 +152,7 @@ module.exports = () => {
 
         expect(runsTable.length).to.equal(4);
 
-        expect(await page.$eval('#runs tbody tr:nth-child(1) td:nth-child(2)', (element) => element.innerText)).to.equal('2FT0,ITS');
+        expect(await page.$eval('#runs tbody tr:nth-child(1) td:nth-child(2)', (element) => element.innerText)).to.equal('2FT0, ITS');
```

Comment:
Space is limited for this particular table, that is why we were displaying detectors without a white space between the elements. Please revert this